setup.sh

```diff
@@ -0,0 +1,7 @@
+#! \bin\bash
+
+apt-get update
+apt-get install -y git python3
+git clone https://github.com/AvshRLev/CS50x-Final-Project/
+apt-get install -y python3-pip
```

Comment:
Any reason this guy is here and not in line 4 with his friends?

Reply:
no.. ill put him with his friends...